jibreen/task.py

```diff
@@ -0,0 +1,78 @@
+
+#using some handling functions from  utils file
+from utils import validate_date,validate_priority,validate_status
+
+# Create module is Task
+class Task:  
+    _id_counter = 1 # Class-level counter for generating unique task IDs
```

Comment:
This approach isn't ideal for Data Persistence because if you rerun the program, the value will reset to 1, even though we might have already saved that value for a different task. You should find a way to address this issue. But anyway, nice work!